scripts/merge-messages.js

```diff
@@ -0,0 +1,18 @@
+const glob = require('glob');
+const fs = require('fs');
+const path = require('path');
+
+const DEST = '.tmp/i18n';
+const CWD = process.cwd();
+
+fs.mkdirSync(DEST, {recursive: true});
+
+glob('i18n/messages*.json', (_, files) => files.forEach(file => {
+  let messages = require(`${CWD}/${file}`);
+  let filename = path.basename(file);
+  let p = require(`${CWD}/node_modules/@nuxeo/nuxeo-ui-elements/i18n/${filename}`);
```

Comment:
Needs to be a path here:
```suggestion
  let p = `${CWD}/node_modules/@nuxeo/nuxeo-ui-elements/i18n/${filename}`;
```